datasets/switchboard.py

```diff
@@ -0,0 +1,293 @@
+"""
+Switchboard is conversational telephony speech with 8 Khz audio files. The training data consists of
+300h hours.
+
+number of recordings: 4876
+number of segments: 249624
+number of speakers: 2260
+"""
+
+
+from sisyphus import *
+
+import glob
+import subprocess
+import shutil
+import os
+import re
+
+from i6_core.lib import corpus
+from i6_core.util import uopen
+
+
+class DownloadSwitchboardTranscriptionAndDictJob(Job):
+    """
+    Downloads switchboard training transcriptions and dictionary (or lexicon)
+    """
+
+    def __init__(self):
+        self.out_trans_and_dict_dir = self.output_path("swb_trans_and_dict")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        zipped_dirname = "switchboard_word_alignments.tar.gz"
+        subprocess.check_call(
+            ["wget", os.path.join("http://www.openslr.org/resources/5", zipped_dirname)]
+        )
+        subprocess.check_call(
+            [
+                "tar",
+                "-xf",
+                zipped_dirname,
+                "-C",
+                ".",
+            ]
+        )
+        shutil.move("swb_ms98_transcriptions", self.out_trans_and_dict_dir)
+
+
+class DownloadSwitchboardSpeakersStatsJob(Job):
```

Comment:
This job is superflous, you can use https://github.com/rwth-i6/i6_core/blob/cef8a57cf0983b810c85c8f5507cc05a34ba4d19/tools/download.py#L12 for this. But if you want you can subclass that job and hardcode the url.

Reply:
this won't work since i need to override the hash function because it checks for checksum. so i will get keyerror. To fix this, i need to override the hash but then i can't call the hash function for the `Job` class right?

Reply:
this works if i pass `checksum=None` to the constructor. i can do this if this is fine.

Reply:
done